app/sass/layouts/_cart.scss

```diff
@@ -0,0 +1,11 @@
+section {
+  @include outer-container;
+
+  #order_items{
+    @include span-columns(6);
+  }
+
+  #order_buttons{
+    @include span-columns(6);
```

Comment:
having not used `bourbon` before, I don’t really know if this is best practice, but is there something like a class which has a column span of 6?